osu.Game/Overlays/Changelog/ChangelogBuild.cs

```diff
@@ -131,33 +131,36 @@ private void load(OsuColour colours, OverlayColourProvider colourProvider)
                         t.Padding = new MarginPadding { Left = 10 };
                     });
 
-                    if (entry.GithubUser.UserId != null)
+                    if (entry.GithubUser != null)
                     {
-                        title.AddUserLink(new User
+                        if (entry.GithubUser.UserId != null)
                         {
-                            Username = entry.GithubUser.OsuUsername,
-                            Id = entry.GithubUser.UserId.Value
-                        }, t =>
-                        {
-                            t.Font = fontMedium;
-                            t.Colour = entryColour;
-                        });
-                    }
-                    else if (entry.GithubUser.GithubUrl != null)
-                    {
-                        title.AddLink(entry.GithubUser.DisplayName, entry.GithubUser.GithubUrl, t =>
+                            title.AddUserLink(new User
+                            {
+                                Username = entry.GithubUser.OsuUsername,
+                                Id = entry.GithubUser.UserId.Value
+                            }, t =>
+                            {
+                                t.Font = fontMedium;
+                                t.Colour = entryColour;
+                            });
+                        }
+                        else if (entry.GithubUser.GithubUrl != null)
                         {
-                            t.Font = fontMedium;
-                            t.Colour = entryColour;
-                        });
-                    }
-                    else
-                    {
-                        title.AddText(entry.GithubUser.DisplayName, t =>
+                            title.AddLink(entry.GithubUser.DisplayName, entry.GithubUser.GithubUrl, t =>
+                            {
+                                t.Font = fontMedium;
+                                t.Colour = entryColour;
+                            });
+                        }
+                        else
                         {
-                            t.Font = fontMedium;
-                            t.Colour = entryColour;
-                        });
+                            title.AddText(entry.GithubUser.DisplayName, t =>
+                            {
+                                t.Font = fontMedium;
+                                t.Colour = entryColour;
+                            });
+                        }
                     }
```

Comment:
This is super ugly but it wasn't pretty before, and refactoring this isn't in scope of this PR, so again, probably going to look away but open an issue for this and the disabled test.